pyrmq/tests/test_consumer.py

```diff
@@ -116,3 +121,36 @@ def error_callback(*args):
 
             wait_for_result(response, [1])
             consumer.close()
+
+
+def should_get_message_with_higher_priority(priority_session: Publisher):
+    for i in range(0, 10):
+        body = {"test": f"test{i}"}
+        if i >= TEST_ARGUMENTS["x-max-priority"]:
+            first_expected = body
```

Comment:
This will be overwritten. In this case, after the loop, first_expected will be the test9.